app/routes/workload-points.js

```diff
@@ -1,11 +1,30 @@
+const workloadPointsService = require('../services/workload-points-service')
+
 module.exports = function (router) {
-  router.get('/workload-points', function (req, res) {
-    return res.render('workload-points', {
-      title: 'Workload Points'
-    })
+  router.get('/admin/workload-points', function (req, res) {
+    var success = req.query.success
+    var successText = success ? 'You have successfully updated the workload points!' : null
+
+    return workloadPointsService.getWorkloadPoints()
+      .then(function (result) {
+        return res.render('workload-points', {
+          title: result.title,
+          subTitle: result.subTitle,
+          breadcrumbs: result.breadcrumbs,
+          wp: result.workloadPoints,
+          successText: successText
+        })
+      })
   })
 
-  router.post('/workload-points', function (req, res, next) {
-    return res.send(req.body.workloadPoints)
+  router.post('/admin/workload-points', function (req, res, next) {
+    var updatedWorkloadPoints = req.body
+
```

Comment:
This is where our server-side validation will need to go, so we might need an intermediate object created from the body. Good to leave for now on the assumption this is covered under the other story in this sprint.

Reply:
👍